packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -660,7 +660,16 @@ - (RCTUIView *)betterHitTest:(CGPoint)point withEvent:(UIEvent *)event // [macOS
   }
 
   for (RCTUIView *subview in [self.subviews reverseObjectEnumerator]) { // [macOS]
-    RCTUIView *hitView = [subview hitTest:[subview convertPoint:point fromView:self] withEvent:event]; // [macOS]
+    // Native macOS views require the point to be in the super view coordinate space for hit testing. [macOS]
```

Comment:
Is this loop macOS specific code? If so, do we still need to mark lines inside this block? And why isn't this loop wrapped in `#if TARGET_OS_OSX`?

Reply:
Good catch, I'll update this.

Reply:
The loop was for iOS too, the macOS tag is because we use `RCTUIView` instead of `UIView`. So actually, this code was mostly sound diff wise. I still updated it to have separate iOS and macOS blocks to make it more clear what we change compared to iOS.